app/services/usps_in_person_proofing/proofer.rb

```diff
@@ -139,13 +147,11 @@ def faraday
       end
     end
 
-    # Retrieve the OAuth2 token (if needed) and then pass
-    # the headers to an arbitrary block of code as a Hash.
-    #
-    # Returns the same value returned by that block of code.
+    # Retrieve and return the set of dynamic headers that are used for most
+    # requests to USPS. An auth token will be retrieved if a valid one isn't
+    # already cached.
+    # @return [Object] Headers to add to USPS requests
```

Comment:
everything is an Object in Ruby, more specifically, it's returning a hash
```suggestion
    # @return [Hash] Headers to add to USPS requests
```